index.js

```diff
@@ -384,30 +351,31 @@ function init (ssb, config) {
       create (opts, cb) {
         const { id: poBoxId, secret } = poBoxKeys.generate()
 
-        onKeystoreReady(() => {
-          keystore.poBox.add(poBoxId, { key: secret }, (err) => {
-            if (err) return cb(err)
+        ssb.box2.addPoBox(poBoxId, { key: secret }, (err) => {
+          if (err) return cb(err)
 
-            cb(null, { poBoxId, poBoxKey: secret })
-          })
+          cb(null, { poBoxId, poBoxKey: secret })
         })
       },
       get: scuttle.group.getPOBox
-    },
+    }
 
     // for internal use - ssb-ahau uses this for backups
-    ownKeys: {
-      list (cb) {
-        onKeystoreReady(() => {
-          cb(null, [keystore.self.get()])
-        })
-      },
-      register (key, cb) {
-        onKeystoreReady(() => {
-          keystore.self.set(key, cb)
-        })
-      }
-    }
+    // TODO: does ahau use this for backups though?
+    // i couldn't find a self.get function in box2 so we might have to add that if this is needed
```

Comment:
@mixmix

ahau didn't seem to use this. i guess ssb-ahau is a different thing, i'll have to check

Reply:
`ownKeys.register` doesn't seem used there but `ownKeys.list` is used here https://gitlab.com/ahau/lib/ssb-plugins/ssb-ahau/-/blob/master/src/graphql/backup.js?ref_type=heads#L50

Reply:
let's make a followup issue for this, since we'll have to update box2

Reply:
made issue https://github.com/ssbc/ssb-tribes/issues/100